### PR TITLE
Allow Bower Plugin to resolve default bower_components directory

### PR DESCRIPTION
**Pull request by antoligy**
Pending lpiepiora/bower-webpack-plugin#27, allow bower webpack plugin to resolve default directory based on .bowerrc.

### CODE REVIEW

**Comment by antoligy** [comment]
Just heard from @lpiepiora that he's on holiday, but that he will review the PR as soon as he's back in 9 days.

**Comment by AlbertoElias** [comment]
Do you guys use a different directory and not bower_components?

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "which": "^1.1.1"
   },
   "devDependencies": {
+    "es6-promise": "^3.0.2",
```
Comment (AlbertoElias) [inline package.json]: obt v4 uses node.js 4 so it doesn't need this

Comment (antoligy) [inline package.json]: Ahh, didn't read that!  Let me go ahead and undo that.

**Comment by antoligy** [comment]
Yep, unfortunately that's a platform limitation- Though it looks like [Puli](http://puli.io/) might soon solve that!

File: lib/tasks/build.js [hunk 2]
```diff
@@ -50,12 +50,23 @@ module.exports.js = function(gulp, config) {
...
 		var dest = config.buildJs || 'main.js';
 
+		var bowerPlugin = new BowerPlugin({
+			includes: /\.js$/
+		});
+
+		// Since (currently) no custom module directories are defined, rely on BowerPlugin dir resolution.
+		var defaultModuleDirectories = bowerPlugin.modulesDirectories;
+		var moduleDirectories = [];
+		defaultModuleDirectories.forEach(function(value) {
```
Comment (AlbertoElias) [inline lib/tasks/build.js]: you can use Array.map and you won't need to create an extra variable

**Comment by AlbertoElias** [comment]
Let's wait and see if your pr is merged in the plugin

**Comment by antoligy** [comment]
Now that #302 has been merged, should I change the destination branch here?

**Comment by AlbertoElias** [comment]
Wait until everything is in master, and then point it to master

**Comment by antoligy** [comment]
Merging master into this without being able to update the destination branch has borked the PR, so closing this in favor of #316.

**Pull request closed without merging**